src/ui/ui_settings/ui_settings_mixin.js

```diff
@@ -0,0 +1,17 @@
+import { UiSettings } from './ui_settings';
+import { mirrorStatus } from './mirror_status';
+
+export function uiSettingsMixin(kbnServer, server, config) {
```

Comment:
__question:__ just curious, is `server.config()` and `config` here is always the same thing? If so why do we pass them separately?